manifests/init.pp

```diff
@@ -195,15 +202,16 @@
     Package['nfs-utils'] -> Package['autofs']
     Service['autofs'] ~> Service[$::nfs::service_names::rpcbind]
 
-    if $::nfs::use_stunnel {
+    if $stunnel {
```

Comment:
This needs to be reverted.

If you're using NFS, the use of Stunnel here is *only* dictated by the use of stunnel in NFS.